docs/conf.py

```diff
@@ -314,6 +323,84 @@ def find_source():
     ),
 ]
 
+
+def _make_estimator_overview(app):
+    """Make estimator overview table."""
+    import pandas as pd
+
+    from aeon.registry import all_estimators
+
+    def _does_not_start_with_underscore(input_string):
+        return not input_string.startswith("_")
+
+    # Columns for the output table
+    COLNAMES = ["Estimator name", "Module", "Family of method"]
+    capabilities_to_include = [
+        "multivariate",
+        "unequal_length",
+        "missing_values",
+    ]
+
+    for capability_name in capabilities_to_include:
+        _str = capability_name.replace("_", " ")
+        COLNAMES.append(f"Support {_str}")
+
+    data = {k: [] for k in COLNAMES}
+
+    for estimator_name, estimator_class in all_estimators():
+        algorithm_type = "::".join(str(estimator_class).split(".")[1:-2])
+        # fetch tags
+        tag_dict = estimator_class.get_class_tags()
+
+        # includes part of class string
+        modpath = str(estimator_class)[8:-2]
+        path_parts = modpath.split(".")
+        # joins strings excluding starting with '_'
+        clean_path = ".".join(list(filter(_does_not_start_with_underscore, path_parts)))
+        # adds html link reference
+        estimator_name_as_link = str(
+            '<a href="https://www.aeon-toolkit.org/en/latest/api_reference'
+            + "/auto_generated/"
+            + clean_path
+            + '.html">'
+            + estimator_name
+            + "</a>"
+        )
+        algorithm_type = algorithm_type.split("::")
+        data["Estimator name"].append(estimator_name_as_link)
+        data["Module"].append(algorithm_type[0])
+        if len(algorithm_type) > 1:
+            data["Family of method"].append("/".join(algorithm_type[1:]))
```

Comment:
Personal preference:
```suggestion
            data["Method family"].append("/".join(algorithm_type[1:]))
```